packages/flutter_localizations/lib/src/l10n/generated_material_localizations.dart

```diff
@@ -140,6 +140,15 @@ class MaterialLocalizationAf extends GlobalMaterialLocalizations {
   @override
   String get invalidDateRangeLabel => 'Invalid range.';
 
+  @override
+  String get licensesPackageDetailTextOne => '1 license';
+
+  @override
+  String get licensesPackageDetailTextOther => '\$licenseCount licenses';
+
+  @override
+  String get licensesPackageDetailTextZero => 'No licenses';
```

Comment:
All of these "Zero" cases would be eliminated if you skip packages with zero licenses.

Reply:
Well, the license page is an inversion of the data hierarchy. LicenseRegistry collects licenses and includes the packages from which those licenses came from as a property on each license. As such, there is a many-to-many relationship between licenses and packages with the constraint that each package has at least one license, and each license has at least one package.

So the zero case is not checked for, since by implementation, it should come up. However, it is undeniably more robust to add checking for, and the exclusion of, packages that have zero licenses. _So I will do this._

However. I would like to highlight, should you include special dispensation for zero license packages, it may be arguable that it is better to highlight packages with zero licenses by using a special unlicensed, or commons, page. But, I also suggest that it is unneeded and zealous work to add this. We should never normally encounter packages with zero licenses. Unless, of course, there is a design change down the line.

Reply:
I just realised why this zero case keeps getting regenerated. It is coming from the default text generated for the different locales from when I ran gen_missing_localizations.dart as instructed to do when adding to material_localizations.